modules/accessing-the-odh-dashboard.adoc

```diff
@@ -0,0 +1,24 @@
+:_module-type: PROCEDURE
+//pv2hash: 
+
+[id='accessing-the-odh-dashboard_{context}']
+= Accessing the Open Data Hub dashboard
+
+[role='_abstract']
+You can access and share the URL for your Open Data Hub dashboard with users to let them log in and work on their data models.
+
+.Prerequisites
+* You have installed the Open Data Hub Operator.
+
+.Procedure
+. In the OpenShift web console, select *Networking* -> *Routes*.
+. Click *Project* and select the *odh* project.
+. Copy the URL in the *Location* column for the *odh-dashboard* route.
```

Comment:
Put the condition first:

"In the Location column, copy the URL for the odh-dashboard route."